lib/modules/datasource/hermit/index.spec.ts

```diff
@@ -0,0 +1,220 @@
+import * as httpMock from '../../../../test/http-mock';
+import type { HermitSearchResult } from './types';
+import { HermitDatasource } from './';
+
+const datasource = new HermitDatasource();
+const githubApiHost = 'https://api.github.com';
+const releaseUrl = '/repos/cashapp/hermit-packages/releases/tags/index';
+const indexAssetUrl = '/repos/cashapp/hermit-packages/releases/assets/38492';
+const sourceAssetUrl = '/repos/cashapp/hermit-packages/releases/assets/38492';
+const registryUrl = 'https://github.com/cashapp/hermit-packages';
+
+describe('modules/datasource/hermit/index', () => {
+  describe('getReleases', () => {
+    it('should return result from hermit list', async () => {
+      const resp = [
+        {
+          Name: 'go',
+          Versions: ['1.17.9', '1.17.10', '1.18', '1.18.1'],
+          Channels: ['@1.17', '@1.18'],
+          CurrentVersion: '1.17.9',
+          Repository: 'https://github.com/golang/golang',
+        },
+      ] as HermitSearchResult[];
```

Comment:
```suggestion
      ];
```

this case will hide type errors